lms/djangoapps/courseware/block_render.py

```diff
@@ -634,9 +635,12 @@ def inner_get_block(block: XBlock) -> XBlock | None:
         'teams_configuration': TeamsConfigurationService(),
         'call_to_action': CallToActionService(),
         'publish': EventPublishingService(user, course_id, track_function),
-        'ai_translation': AiTranslationService(),
     }
 
+    # Make AI translations service available for course, only where enabled
+    if whole_course_translations_enabled_for_course(course_id):
```

Comment:
Does `course_id` differ from `course_key`?

Reply:
Conventionally we use these to refer to slightly different objects, but here `course_id` works for the `whole_course_translations...` function.